### DISCUSSION TITLE
Event not received

### DISCUSSION THREAD

**Discussion by DrAnax** [qruto/laravel-wave#42] [General]
I'm trying laravel-wave locally, using Laravel 10 and sail. My front-end script looks like this:

```
import Echo from 'laravel-echo';
import { WaveConnector } from 'laravel-wave';
window.Echo = new Echo({ 
    broadcaster: WaveConnector,
 });

console.log('listening');

window.Echo.channel('events')
    .listen('RealTimeNotification', (e) => {
        console.log(e.message);
    });
```

I also have created a notification like this:

```
<?php

namespace App\Events;

use Illuminate\Broadcasting\Channel;
use Illuminate\Broadcasting\InteractsWithSockets;
use Illuminate\Contracts\Broadcasting\ShouldBroadcast;
use Illuminate\Foundation\Events\Dispatchable;
use Illuminate\Queue\SerializesModels;

class RealTimeNotification implements ShouldBroadcast
{
    use Dispatchable, InteractsWithSockets, SerializesModels;

    public function __construct(public string $message)
    {
    }

    public function broadcastOn()
    {
        return new Channel('events');
    }
}

```

when I fire a new notification via tinker (`event(new \App\Events\RealTimeNotification("test"))`) nothing happens. I can see the queue processing the notification, I can also see network activity (must be pings) with status 200 on the browser console. Any ideas?

**Comment by slavarazum** [comment]
Hey 👋 

Try to enable [debug mode](https://github.com/qruto/laravel-wave#client-options) to get more detailed information in the console.

**Comment by DrAnax** [comment]
I have; the message never gets to the client

**Reply by DrAnax** [reply]
Given I used laravel-wave in an existing application, my best guess is that there's something on my pre-existing code; I'll let you know.

**Reply by DrAnax** [reply]
I noticed that in the vanilla installation with debug mode enabled I see two events: `general.connected` and `general.ping` whereas in my application I only see `general.connected`. Does that ring any bells?

**Reply by slavarazum** [reply]
Have you tried to run ping manually?

```bash
php artisan sse:ping
```

Do you use any queue driver for sending event? If yes, try to test it with the `sync` driver.

**Reply by DrAnax** [reply]
Everything works as expected. My application runs on docker locally; hence the problems. I was running queue:work on the wrong terminal.
